src/seedu/addressbook/data/person/Contact.java

```diff
@@ -0,0 +1,37 @@
+package seedu.addressbook.data.person;
+
+/**
+ * Represents a Person's contract in the address book.
```

Comment:
Typo detected :-P *contact* not *contract*,
Secondly, no need to mention it as in address book, `Contact` can be used in any application requiring to store the contact information